rtf_vis_tool/src/Components/SfMDataDivNode.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+//The Gray 'SfMData' Node Component 
+//When clicked, it spawns the data association point cloud (located in Data_Association_PC.js)
+const SfMDataDivNode = (props) => {
+
+    //similar styling applies to DivNode.js
+    const divNodeStyle = {
```

Comment:
can we import the same variable/style? If two things have to be the same, I'd suggest storing just a single copy.

Reply:
created the CSS file to store most of the duplicate stylings.